### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by hmadison**
Sets up services needed for test.

### CODE REVIEW

**Comment by talklittle** [comment]
Awesome! Maybe add the badge to the README while you're at it?

    [Build Status](https://travis-ci.org/lobsters/lobsters)

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     expect { User.make!(:username => "") }.to raise_error
     expect { User.make!(:username => "*") }.to raise_error
 
-    User.make!(:username => "test")
+    User.make!(:username => "test") unless User.find_by(username: 'test')
```
Comment (pushcx) [inline spec/models/user_spec.rb]: Why is this needed? Is this papering over a test failing to clean the database?

**Comment by pushcx** [comment]
Thanks, this is great to have on PRs.

File: spec/models/user_spec.rb (hunk 1 above)
Comment (hmadison) [reply spec/models/user_spec.rb]: This fixed a failure on two runs:
* https://travis-ci.org/lobsters/lobsters/builds/283332809?utm_source=github_status&utm_medium=notification
* https://travis-ci.org/lobsters/lobsters/builds/283330930?utm_source=github_status&utm_medium=notification

We have transnational fixtures turned on, so we should be clearing the database after each run.

Comment (pushcx) [reply spec/models/user_spec.rb]: Uh... that sure looks like either test data failed to get cleaned or some general pre-test hook is creating duplicate data. There's something wrong here, this `unless` shouldn't exist. I'm sorry I didn't think of this before hitting merge.

Comment (hmadison) [reply spec/models/user_spec.rb]: Ill dig at this more locally and try to see whats causing the issue.

**Comment by pushcx** [comment]
Reopening so I can't forget about the `unless` again. @hmadison, please reuse this PR if you can land a fix on your `travis--conf` branch. Hopefully GitHub does the right thing.

File: .travis.yml [hunk 2]
```diff
@@ -1,2 +1,9 @@
 ---
 language: ruby
+services:
+  - mysql
+env:
+ - DATABASE_URL=mysql2://root:@localhost/lobsters_test
```
Comment (talklittle) [inline .travis.yml]: nitpick: indentation :)

**Comment by hmadison** [comment]
@pushcx The issue was the seeds.rb file had a definition for the test user which was getting loaded in before the test run.

**Comment by pushcx** [comment]
Great, thank you for running that down!

**Pull request merged**